alg/gdaltransformer.cpp

```diff
@@ -547,9 +554,9 @@ GDALSuggestedWarpOutput2( GDALDatasetH hSrcDS,
                     iSignDiscontinuity = SIGN_FINAL_INVALID;
                 }
             }
-            else if( iSignDiscontinuity == SIGN_FINAL_UNINIT )
+            else if( iSignDiscontinuity )
```

Comment:
please revert that change. it is too smart and relies on the actual value of SIGN_FINAL_UNINIT